docker/docker-compose.yml

```diff
@@ -71,8 +71,17 @@ services:
   processor:
     image: context-processor:0.1.1
     environment:
-      KAFKA_BROKER: "kafka1:29092"
+      KAFKA_BROKER: "host.docker.internal:29092"
```

Comment:
It is weird this is required on yours and not mine. The interface really should be kafka1, at least according to my experience; I guess we'll find out :)

Reply:
I'll try it as kafka1. I changed this when I was trying to debug an issue. It is also working with kafka1 so I'll update this.